documentation/ipi-install/modules/ipi-install-creating-a-disconnected-registry.adoc

```diff
@@ -198,11 +198,13 @@ Replace `<username>` with the user name and `<passwd>` with the password.
 NOTE: Example output of the variables used to mirror the install images
 +
 ----
-UPSTREAM_STREAM=${RELEASE_IMAGE}
+UPSTREAM_REPO=${RELEASE_IMAGE}
 LOCAL_REG=<registry_FQDN>:<registry_port>
 LOCAL_REPO='ocp4/openshift4'
 ----
 
+(Recall that we set `RELEASE_IMAGE` and `VERSION` in xref:ipi-install-retrieving-openshift-installer.adoc[Retrieving OpenShift Installer])
```

Comment:
I'd say remove the quotes around (Recall...)

Also the xref you are looking for is

xref:retrieving-openshift-installer_{context}[Retrieving OpenShift Installer]

However, this id doesn't exist for the file. Can you include this 
```
[id="retrieving-openshift-installer_{context}"]
```
in file modules/ipi-install-retrieving-openshift-installer.adoc ?

It should go right above the title and look like such:

```
// Title to include both UPS and GA
[id="retrieving-openshift-installer_{context}"]
= Retrieving OpenShift Installer

```